roles/mysql-5.7/meta/main.yml

```diff
@@ -12,6 +12,7 @@ dependencies: # noqa 701
  - role: collectd
    collectd_deployed_app_name: "mysql"
    collectd_deployed_app_version: "{{ mysql_server_version }}"
+   collectd_version: "{{ server_collectd_version }}"
```

Comment:
Should the value not be from `mysql_collectd_version` instead? It seems to be going against how variables are named in the role.

Reply:
updated